test/helpers/web3/initializeTestServer.js

```diff
@@ -0,0 +1,28 @@
+const Ganache = require(process.env.TEST_BUILD
+  ? "../../../build/ganache.core." + process.env.TEST_BUILD + ".js"
+  : "../../../index.js");
+const Web3 = require("web3");
+
+const intiailizeTestServer = (tests, port = 12345) => {
```

Comment:
I don't think this file should be here. Also, `initialize` is spelled wrong. :-D

Reply:
Where do you think it should go?